## Rank top 3 Breweries which produce the strongest beers?

In [1]:
import pandas as pd

In [32]:
df = pd.read_csv('data/BeerDataScienceProject.csv', engine='python')
df.shape

df.head(2)

beer_ABV  beer_beerId  beer_brewerId     beer_name          beer_style  \
0       5.0        47986          10325  Sausa Weizen          Hefeweizen   
1       6.2        48213          10325      Red Moon  English Strong Ale   

   review_appearance  review_palette  review_overall  review_taste  \
0                2.5             2.0             1.5           1.5   
1                3.0             2.5             3.0           3.0   

  review_profileName  review_aroma  \
0            stcules           1.5   
1            stcules           3.0   

                                         review_text  review_time  
0  A lot of foam. But a lot. In the smell some ba...   1234817823  
1  Dark red color, light beige foam, average. In ...   1235915097

In [33]:
df.sort_values(['beer_ABV'], ascending=False).drop_duplicates(['beer_beerId'], keep='first')[:3]

beer_ABV  beer_beerId  beer_brewerId                      beer_name  \
12889     57.70        73368           6513  Schorschbr�u Schorschbock 57%   
12909     43.00        57856           6513  Schorschbr�u Schorschbock 43%   
12911     39.44        55712           6513  Schorschbr�u Schorschbock 40%   

      beer_style  review_appearance  review_palette  review_overall  \
12889    Eisbock                4.0             4.0             4.0   
12909    Eisbock                3.5             4.0             4.0   
12911    Eisbock                3.0             3.0             3.0   

       review_taste review_profileName  review_aroma  \
12889           4.0        kappldav123           3.5   
12909           4.0           Sunnanek           4.0   
12911           2.0          orangesol           3.0   

                                             review_text  review_time  
12889  New world strongest beer. Very limited, only 3...   1316780901  
12909  43% ABV - this is so crazy! A: Pour is warm am...   1309974178  
12911  Thanks to foofaa for sharing his sample with m...   1321041551

## Top 3 strongest beers:
- Schorschbr�u Schorschbock 57%
- Schorschbr�u Schorschbock 43%
- Schorschbr�u Schorschbock 40%